marklogic-data-hub-central/ui/src/util/search-context.tsx

```diff
@@ -385,7 +389,7 @@ const SearchProvider: React.FC<{ children: any }> = ({ children }) => {
     });
   }
 
-    const applySaveQuery = (searchText: string, entityTypeIds: string[], selectedFacets: {}, selectedQuery: string, zeroState = false, manageQueryModal = false) => {
+    const applySaveQuery = (searchText: string, entityTypeIds: string[], selectedFacets: {}, selectedQuery: string, propertiesToDisplay: string[], zeroState = false, manageQueryModal = false) => {
```

Comment:
I would consider refactoring the variables into an options object. You can create an interface that goes with it, and share it across components that call the applySaveQuery function.

Reply:
Thanks, will implement interface in next story.